pyxs/exceptions.py

```diff
@@ -51,6 +51,10 @@ class InvalidPath(ValueError, PyXSError):
     :param bytes path: invalid path value.
 """
 
+class NonexistentPath(ValueError, PyXSError):
+    """
+    Exception raised when a path is not an actual Xenstore path.
```

Comment:
I know this is small, but consistency matters. We use XenStore (not the capital S) through out documentation.